webpack/assets/javascripts/react_app/components/hosts/storage/vmware/controller/index.js

```diff
@@ -112,7 +105,7 @@ const Controller = ({
           <Select
             value={controller.type}
             disabled={config.vmExists}
-            onChange={e => _updateController('type', e)}
+            onValueChange={v => _updateController('type', v)}
```

Comment:
imo it's more readable as:
```js
onValueChange={value => updateController({ type: value })}
```

Reply:
the helper function is actually translating it like that, so i have deleted the helper function and did this right away, thanks :+1: